src/main/scala/com/advancedtelematic/campaigner/data/DataType.scala

```diff
@@ -20,7 +20,7 @@ object DataType {
 
   object MetadataType extends Enumeration {
     type MetadataType = Value
-    val install = Value
+    val description, estimatedInstallationDuration, estimatedPreparationDuration = Value
```

Comment:
How do we feel about this capitalization after all?

Reply:
I do like capitalization for enums. But do not forget to adjust the DB schema too

Reply:
Oh I see now about my previous PR. Alright, it's capitalized now.